tenable/io/v3/groups.py

```diff
@@ -0,0 +1,184 @@
+"""
+Groups
+======
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`groups <groups>` API.
+
+Methods available on ``tio.v3.groups``:
+
+.. rst-class:: hide-signature
+.. autoclass:: GroupsAPI
+    :members:
+"""
+from typing import Dict, List
+from uuid import UUID
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class GroupsAPI(ExploreBaseEndpoint):
+    """
+    This will contain all methods related to Groups
+    """
+
+    _path = "api/v3/groups"
+    _conv_json = True
+
+    def add_user(self, group_id: UUID, user_id: UUID) -> None:
```

Comment:
I think we should get back the Group object as a response once it has been updated

Reply:
Done. As we don't have an API to fetch the details of a particular Group, we are returning the list of users present in the specified Group ID. Refer: line [#52](https://github.com/dhvanilshah-tenable/pyTenable/pull/9/files#diff-987e96ee82ce6b6085aaf7e527916eb78fb2846da1b3a5e7a8a16dba27a6ed22R52).